lib/store_web/templates/index/index.html.leex

```diff
@@ -61,14 +61,16 @@
         <li>
           <%= live_patch to: Routes.product_path(Endpoint, :index, product.id) do %>
             <img alt="<%= product.name %>" loading="lazy" src="<%= product.thumbnail_url %>" />
-            <h3><%= product.name %></h3>
-            <h4>
-              <%= if Enum.at(product.price_range, 0) == Enum.at(product.price_range, 1) do %>
-                $<%= product.price_range |> Enum.at(0) |> format_price() %>
-              <% else %>
-                $<%= product.price_range |> Enum.at(0) |> format_price() %>+
-              <% end %>
-            </h4>
+            <h3>
+              <%= product.name %>
+              <span class="price">
+                <%= if Enum.at(product.price_range, 0) == Enum.at(product.price_range, 1) do %>
+                  $<%= product.price_range |> Enum.at(0) |> format_price() %>
+                <% else %>
+                  $<%= product.price_range |> Enum.at(0) |> format_price() %>+
+                <% end %>
```

Comment:
This might make this a bit cleaner to read

```suggestion
                $<%= product.price_range |> Enum.at(0) |> format_price() %>
                <%= if not Enum.at(product.price_range, 0) == Enum.at(product.price_range, 1) do %>+<% end %>
```